packages/core/src/generators/import-projects/generator.ts

```diff
@@ -71,10 +70,6 @@ async function addNewDotnetProject(
     configuration.targets.test = GetTestExecutorConfig();
   }
   addProjectConfiguration(host, projectName, configuration);
-  await manipulateXmlProjectFile(host, {
```

Comment:
Should `init` generator be run as part of `import-projects` with the removal of this XML manipulation?

Reply:
Yeah, `init` **_should_** have always been called in here, that's a miss on my earlier part

Reply:
Ok, I added the call of the `init` generator without conditions, so it is run even if no projects are found.

Also, I put 
```typescript
return async () => {
  await installTasks()
}
```
and it's inconsistent with `GenerateProject`, but I wasn't sure if the not awaited result there is intentional.